src/popup/components/ConnectWalletForm.tsx

```diff
@@ -62,34 +65,34 @@ export const ConnectWalletForm = ({
   const [recurring, setRecurring] = React.useState<Inputs['recurring']>(
     defaultValues.recurring || false,
   );
+
   const [autoKeyShareFailed, setAutoKeyShareFailed] = React.useState(
     isAutoKeyAddFailed(state),
   );
 
   const resetState = React.useCallback(async () => {
     await clearConnectState();
-    setErrors((_) => ({ ..._, keyPair: '', connect: '' }));
+    setErrors((_) => ({ ..._, keyPair: null, connect: null }));
```

Comment:
I suggest renaming the parameter to `prev` or something else that suggests that this is the previous state. Clearer than `_`.
```suggestion
    setErrors((prev) => ({ ...prev, keyPair: null, connect: null }));
```